core-impl/proxy/src/test/java/com/alipay/sofa/rpc/proxy/bytebuddy/BytebuddyProxyTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.proxy.bytebuddy;
+
+import com.alipay.sofa.rpc.core.request.SofaRequest;
+import com.alipay.sofa.rpc.log.Logger;
+import com.alipay.sofa.rpc.log.LoggerFactory;
+import com.alipay.sofa.rpc.proxy.AbstractTestClass;
+import com.alipay.sofa.rpc.proxy.TestInterface;
+import com.alipay.sofa.rpc.proxy.TestInvoker;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+/**
+ * @author bystander
+ * @version $Id: BytebuddyTest.java, v 0.1 2019年01月30日 11:16 bystander Exp $
+ */
+public class BytebuddyProxyTest {
+
+    private final static Logger LOGGER = LoggerFactory.getLogger(BytebuddyProxyTest.class);
+
+    @Test
+    public void getProxy() throws Exception {
+        BytebuddyProxy proxy = new BytebuddyProxy();
+        AbstractTestClass testClass = null;
+        try {
+            testClass = proxy.getProxy(AbstractTestClass.class, new TestInvoker());
+        } catch (Exception e) {
+            LOGGER.info(e.getMessage());
+        }
+        Assert.assertNotNull(testClass);
```

Comment:
Wrong interface class still can get proxy class?

Reply:
yes, bytebuddy support abstract class. sofa will prevent this.but as an proxy,we do not need to.

Reply:
OK